packages/jest-mock-scheduler/package.json

```diff
@@ -20,7 +20,7 @@
   "homepage": "https://reactjs.org/",
   "peerDependencies": {
     "jest": "^23.0.1",
-    "scheduler": "^0.11.0"
```

Comment:
Should we really bump this one?

Reply:
Was it intentional that we didn't raise it past 0.11 previously? Or was that a mistake?

Reply:
The API changed in 0.14 so yeah. It's private anyway so doesn't really matter.